nixos/modules/config/shells-environment.nix

```diff
@@ -198,6 +206,11 @@ in
           # ~/bin if it exists overrides other bin directories.
           export PATH="$HOME/bin:$PATH"
         ''}
+
+        ${optionalString cfg.localBinInPath ''
+          # ~/.local/bin if it exists overrides other bin directories.
+          export PATH="$HOME/.local/bin:$PATH"
```

Comment:
Should the `L` be capital in `.local`?

Reply:
why would it be capital.. FHS3_0 defines that as `.local` and nixOS seems to be using it?

![image](https://user-images.githubusercontent.com/11302521/123532198-89b6c200-d70b-11eb-9ca2-80f7812499a1.png)

Reply:
my bad, it seems i confuse the L font in github with the l font.